app/views/analytics/districts/show.csv.erb

```diff
@@ -77,8 +83,8 @@ end) %>
 <% csv << [
     "All",
     nil,
-    @dashboard_analytics.sum { |_, row| row[:total_registered_patients] || 0 },
-    *dates.map { |period| analytics_totals(@dashboard_analytics, :follow_up_patients_by_period, period) },
+    @dashboard_analytics.sum { |_, row| row[:total_patients] || 0 },
+    *dates.map { |period| analytics_totals(@dashboard_analytics, :patients_with_bp_by_period, period) },
```

Comment:
These data structure changes are taken care of in another PR, right?

Reply:
Yup, it's a little odd to look at this PR without that context, but the changes are split in 2 so I couldn't base off of that.